dracut/python-deps

```diff
@@ -70,6 +70,10 @@ except AttributeError:
 while scripts:
     script = scripts.pop()
 
+    if script == 'frozen':
```

Comment:
Should this rather be “if `script` doesn't name an existing file”?

Reply:
Yes. But with the other cases, do we know we can safely skip them?